legend-engine-language-pure-store-relational/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/connection/authentication/AuthenticationStrategyLexerGrammar.g4

```diff
@@ -20,6 +20,10 @@ SNOWFLAKE_AUTH_KEY_VAULT_REFERENCE:         'privateKeyVaultReference';
 SNOWFLAKE_AUTH_PASSPHRASE_VAULT_REFERENCE:  'passPhraseVaultReference';
 SNOWFLAKE_AUTH_PUBLIC_USERNAME:             'publicUserName';
 
+USER_PASSWORD:                              'UserPassword';
+USER_NAME:                                  'userName';
```

Comment:
Nitpick. Case for user password and user name is inconsistent.

Reply:
User Password is the name of the auth strategy here